packages/jupyter/src/embed.tsx

```diff
@@ -0,0 +1,66 @@
+import { SourceFileKind, type GenericNode } from 'myst-common';
+import { useCellExecution } from './execute';
+import {
+  ArticleResetNotebook,
+  ArticleRunNotebook,
+  ArticleStatusBadge,
+} from './controls/ArticleCellControls';
+import { JupyterIcon } from '@scienceicons/react/24/solid';
+import { select } from 'unist-util-select';
+import { useLinkProvider } from '@myst-theme/providers';
+
+function EmbedWithControls({
```

Comment:
I have moved the `embed -> output` node with controls here. This allows us to link to the notebook source.

I am not totally happy with it, but I think it is better than the controls being fully on the right.